Library/Homebrew/cask/audit.rb

```diff
@@ -246,7 +246,7 @@ def check_download_url_format
     end
 
     def bad_url_format?(regex, valid_formats_array)
-      return false unless cask.url.to_s =~ regex
+      return false unless cask.url.to_s&.match?(regex)
```

Comment:
`&` isn't needed here.

Reply:
Clearly Rubocop's autocorrection is fallible. :-) I'll fix them all.